src/gateway/router/routes/deployContractRoute.ts

```diff
@@ -113,7 +113,7 @@ export async function deployContractRoute(ctx: Router.RouterContext) {
         src_wasm_lang: srcWasmLang || null,
         bundler_src_tx_id: bundlerSrcTxId,
         bundler_src_node: BUNDLR_NODE2_URL,
-        bundler_response: JSON.stringify(srcBundlrResponse.data),
+        bundler_response: JSON.stringify(srcBundlrResponse?.data),
```

Comment:
it doesn't compile without this change:
<img width="823" alt="image" src="https://user-images.githubusercontent.com/84311757/204036191-bf30147d-f1b5-4bb6-bc03-c3ad93102e89.png">

not sure how it worked before...